src/lib/toCamelCase.ts

```diff
@@ -0,0 +1,11 @@
+const toCamelCase = (snakeCase) => {
```

Comment:
It's not very likely to break I guess but a unit test would not hurt

Reply:
thanks for the push